### ISSUE TITLE
Thickmuck keep terrain trap

### ISSUE THREAD

**Issue by Simmox** [WarEmu/WarBugs#8137]
i can probably get a screenshot if required...........................

thickmuck keep badlands
we are inside the outer door and looking at the actual keep door .so
to the right hand side of the right hand front tower on the outside,there is a medium large skeletal rib cage on the ground ,it looks like a proper cage
i was knock backed towards this rib cage and was stuck inside it,only a book bind got me out
so in short,SE corner of the keep,the rib cage there can trap players

outrageous:)

**Comment by Luth-RoR** [comment]
A client side issue; can't be fixed yet.
